compiler/main/driver.cpp

```diff
@@ -149,6 +144,7 @@ int scalar_replace_limit = 8;
 int tuple_copy_limit = scalar_replace_limit;
 bool fGenIDS = false;
 int fLinkStyle = LS_DEFAULT; // use backend compiler's default
+bool fSetLocal = false;
 bool fLocal;   // initialized in setupOrderedGlobals() below
```

Comment:
There's some comments referring to setupOrderedGlobals() that are out of date now